src/components/Checkbox/index.tsx

```diff
@@ -104,6 +104,7 @@ const CheckboxContainer = styled.label`
   position: relative;
   display: inline-flex;
   align-items: center;
+  gap: ${({ theme }) => theme.space['1']};
```

Comment:
Meaning we can remove the margin-right on StyledActivityContainer too :)

Reply:
Good catch !
Fixed.